docs/src/content/docs/guides/customization.mdx

```diff
@@ -0,0 +1,411 @@
+
+To enable edit links, set [`editLink`](/reference/configuration/#editlink) in the Starlight integration config.
+The value of `editLink.baseUrl` will be prepended to the path to the current page to form the full edit link.
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+export default defineConfig({
+  integrations: [
+    starlight({
+      title: 'Docs With Edit Links',
+      editLink: {
+        baseUrl: 'https://github.com/withastro/starlight/edit/main/',
+      },
+    }),
+  ],
+});
+```
+
+## Custom CSS styles
+
+Customize the styles applied to your Starlight site by providing additional CSS files to modify or extend Starlight’s default styles.
+
+1. Add a CSS file to your `src/` directory.
+   For example, you could override Starlight’s default blue accent hue to purple:
+
+   ```css
+   /* src/styles/custom.css */
+   :root {
+     --sl-hue-accent: 270;
+   }
+   ```
+
+2. Add the path to your CSS file to Starlight’s `customCss` array in `astro.config.mjs`:
+
+   ```js
+   // astro.config.mjs
+   import { defineConfig } from 'astro/config';
+   import starlight from '@astrojs/starlight';
+
+   export default defineConfig({
+     integrations: [
+       starlight({
+         title: 'Docs With Custom CSS',
+         customCss: [
+           // Path to your custom CSS file, starting with /
+           '/src/styles/custom.css',
+         ],
+       }),
+     ],
+   });
+   ```
+
+## Custom fonts
+
+By default, Starlight uses sans-serif fonts available on a user’s local device for all text.
+This ensures documentation loads quickly in a font that is familiar to each user, without requiring extra bandwidth to download large font files.
+
+If you do need to add a custom font to your Starlight site, this guide will show you how.
+
+### Set up fonts
+
+If you already have font files, follow the [local set-up guide](#set-up-local-font-files).
+If you are considering using Google Fonts, we recommend following the [Fontsource set-up guide](#set-up-a-fontsource-font).
+
+#### Set up local font files
+
+1. Add your font files to a `src/fonts/` directory and create an empty `font-face.css` file:
+
+   <FileTree>
+
+   - src/
+     - content/
+     - fonts/
+       - **CustomFont.woff2**
+       - **font-face.css**
+   - astro.config.mjs
+
+   </FileTree>
+
+2. Add an [`@font-face` declaration](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face) for each of your fonts in `src/fonts/font-face.css`.
+   Use a relative path to the font file in the `url()` function.
+
+   ```css
+   /* src/fonts/font-face.css */
+
+   @font-face {
+     font-family: 'Custom Font';
+     /* Use a relative path to the local font file in `url()`. */
+     src: url('./CustomFont.woff2') format('woff2');
+     font-weight: normal;
+     font-style: normal;
+     font-display: swap;
+   }
+   ```
+
+3. Add your `@font-face` declarations to Starlight’s `customCss` array in `astro.config.mjs`:
```

Comment:
```suggestion
3. Add the path to your `font-face.css` file to Starlight’s `customCss` array in `astro.config.mjs`:
```
You're not adding the declarations directly (as in, what you're typing into config here), so it's probably clearer to say "add the file path to the file with your declarations" (you can add "CSS file *containing the `@font-face declarations`* to Starlight's ..."  if you think that's helpful, but I think it suffices to say add the path in config.)

Reply:
Yes, I think this is better as-is!